Makefile

```diff
@@ -226,11 +227,15 @@ monitoring_start: docker_check
 docker_loki_install: docker_check
 	docker plugin install grafana/loki-docker-driver:latest --alias loki --grant-all-permissions
 
+clean_mocks:
+### Use `clean_mocks` delete mocks before recreating them. Also useful to cleanup code that was generated from a differen branch
+	$(eval modules_dir = "shared/modules")
+	find ${modules_dir}/mocks -maxdepth 1 -type f ! -name "mocks.go" -exec rm {} \;
```

Comment:
Why maxdepth 1?

Reply:
Genuine mistake. I started this with "how do I delete everything but one specific file in one folder" and I carried too far the "in one folder" part.

Currently not an issue since we put mocks only at the root but without `maxdepth` it should be more future-proof if we change things later on.